From daefa1eb9e9e83cb29460ba37e32ccb22b7a82e3 Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Fri, 26 Sep 2025 10:30:54 +0200
Subject: [PATCH 221/553] Mark qformat_impl.h as security-significant

String formatting is, of course, security-critical, but this file
contains no code, so picked headers-decl-only as the closest reason.

Not picking to 6.8, because this file (incl. its predecessor from io/)
was only added for 6.9.

Task-number: QTBUG-135195
Change-Id: I482775f4dba70bdd70fea4581923042971a0d196
Reviewed-by: Ivan Solovev <ivan.solovev@qt.io>
(cherry picked from commit df01d275f0669891941d44caa3d4ef2fd838be90)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/text/qtformat_impl.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/corelib/text/qtformat_impl.h b/src/corelib/text/qtformat_impl.h
index d9ab96a3b0f..385086ef383 100644
--- a/src/corelib/text/qtformat_impl.h
+++ b/src/corelib/text/qtformat_impl.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2024 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:header-decls-only
 
 #ifndef QTFORMAT_IMPL_H
 #define QTFORMAT_IMPL_H
-- 
2.51.2

